net.launchpad.Gnuclad.yml

```diff
@@ -0,0 +1,16 @@
+app-id: net.launchpad.Gnuclad
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: gnuclad
+finish-args:
+  - --filesystem=home
+modules:
+  - name: gnuclad
+    buildsystem: simple
+    build-commands:
+      - install -D gnuclad /app/bin/gnuclad
+    sources:
+      - type: file
+        url: https://github.com/eylenburg/gnuclad-snap/raw/main/bin/gnuclad
```

Comment:
I am not sure how this is supposed to work:

1. there is not even any display permission
2. there is no appstram file, icon, and desktop file installed
3. you install what I assume is a binary but still try to build a flatpak for aarch664 and x86_64
4. why isn't this pulled from upstream anyway and build from source?

Reply:
Regarding 1 and 2: It's a CLI app, doesn't need display permission, icon or desktop file. I only wanted to build this as a Flatpak so it can be used in an immutable distro.

Regarding 3 and 4: That's a valid concern. I got a compilation error, both when trying to create a Flatpak or when trying to compile it as a normal application, and I couldn't figure it and have no tech background, so I just decided to use the binary. It's a very simple application with only that one binary and no further dependencies. The only downside I guess is that the binary only works on x86_64

Reply:
appstream file is required

also flatpak is not ideal for CLI application as they are not shown anywhere. 

out of the box this won't build.

Reply:
I know that Flatpak is mainly for GUI applications, and I actually packaged this as a Snap already, but I personally prefer Flatpak and didn't find anywhere saying you can't have CLI applications on Flathub.

What would be the requirements to get this on Flathub? Add an Appstream file and link the YAML file to the source code on launchpad? Or do you not want CLI applications in general?